### ISSUE TITLE
GUI issues in Pantheon

### ISSUE THREAD

**Issue by jasonphi** [racket/drracket#126]
Using Pantheon DE, a number Dr Racket windows are broken. The most significant window being the "Choose Language" dialog, making Dr Racket very difficult to use. [Another user had a similar experience on Reddit](https://www.reddit.com/r/elementaryos/comments/6de4q2/dr_racket_gui_not_working_properly/) (though I could not find any other issues filed)

![drracket splash](https://user-images.githubusercontent.com/29696426/27521567-898284ae-59ec-11e7-9212-9ae869c54f36.png)

![drracket choose language](https://user-images.githubusercontent.com/29696426/27521570-8c1c080c-59ec-11e7-8ee5-55818534304f.png)

**Comment by redtrumpet** [comment]
I have the same problem.

This can be fixed, by disabling GTK client side decoration, e.g. opening DrRacket, by doing

    GTK_CSD=""
    drracket

in a terminal or a start script.

But this workaround is not really user friendly and the problem should still be considered a bug.

**Comment by spdegabrielle** [comment]
Is Pantheon the window manager on ArchLinux?

**Comment by spdegabrielle** [comment]
Hi @jasonphi / @redtrumpet 
First let me start by saying I'm not part of the racket team so I'm not speaking on their behalf. 
If this is an important issue for you that you would like to resolve quickly *my* suggestion is to look through the racket github repository for the scripts that build the start script for arch linux, and submit a pull request to add the fix that @redtrumpet identified
GTK_CSD=""

HTH 
Stephen

**Comment by redtrumpet** [comment]
> Is Pantheon the window manager on ArchLinux?

I don't think so, it's the window manager / desktop environment of [elementary OS](https://elementary.io).  I'm no Arch expert, but if I understand Arch correctly, it does not have a canonical window manager, but you can install whatever you like.

I'm in no need for a quick fix, as I don't really use DrRacket. I just wanted to try installing it (used it a few years ago in school), and run into this issue. I thought it was worth reporting my fix.

I also don't know if `GTK_CSD=""` breaks some things on other systems / desktop environments, so it might not be safe changing the build scripts? But I'm just guessing...

**Comment by rajkumaar23** [comment]
This issue still exists and I would love to see if this gets fixed :/ 

I'm not even able to close the application :(
![image](https://user-images.githubusercontent.com/37476886/88505058-fc975480-cff3-11ea-905a-49ae60c704db.png)
